src/helpers.js

```diff
@@ -316,6 +380,13 @@ export function collides(obj1, obj2) {
  */
 export function getWorldRect(obj) {
   let { x = 0, y = 0, width, height } = obj.world || obj;
+  let radius = obj.world.radius || obj.radius;
+
+  // circles
+  if (radius) {
+    width = radius;
```

Comment:
Width and height should be twice the radius.

Reply:
😬